README.md

```diff
@@ -0,0 +1,229 @@
+  <summary>Style</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Linters</summary>
+  <ul>
+    <li><a href="https://webhint.io/">Webhint</a></li>
+    <li><a href="https://stylelint.io/">Stylelint</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **[Linter]** Linter is a tool to help you improve your code, it analizes the source code looking for problems.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](NA)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+  
+  
+### Prerequisites
+
+In order to run this project you need:
+  Open terminal on the same folder of the project and run:
+```sh
+ npm install
+```
+
+
+### Setup
+
+Clone this repository to your desired folder:
+```sh
+  cd my-folder
+  git clone git@github.com/codedit334/Hello_Microverse.git
+```
+
+
+### Install
+
+Install this project with:
+```sh
+  cd Hello_Microverse
+  npm install
+```
+
+
+### Usage
+
+To run the project, open index.html
+
+
+### Run tests
+
+To run tests, run the following command:
+
+To check for html errors run:
+```sh
+  npx hint .
+```
+
+To check for css errors run:
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+
+### Deployment
+
+You can deploy this project using your preferd hosting platform.
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/codedit334)
+- Twitter: [@twitterhandle](https://twitter.com/oussedit)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/oussama-elabdioui-4677a41b6/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[Full HTML5 page]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/codedit334/Hello_Microverse/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
```

Comment:
- [ ] Kindly consider removing the description of this section of your ReadMe to improve its professionalism. This will detail your documenting skills.
- [ ] [OPTIONAL]! Also, I think the ReadMe would be a lot more descriptive if you avoid writing/making incomplete sentences like the one on line 204. :wink: